File: mobile_app/app/Objects/src/com/menube/app/designerscripts/LS_frmreporteordenes.java
package com.menube.app.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmreporteordenes{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
views.get("panelordenes").vw.setLeft((int)(0d));
views.get("panelordenes").vw.setWidth((int)((100d / 100 * width) - (0d)));
views.get("panelordenes").vw.setTop((int)(0d));
views.get("panelordenes").vw.setHeight((int)((100d / 100 * height) - (0d)));
views.get("panelordentitulo").vw.setLeft((int)(0d));
views.get("panelordentitulo").vw.setTop((int)(0d));
views.get("panelordentitulo").vw.setHeight((int)((8d / 100 * height)));
views.get("panelordentitulo").vw.setWidth((int)((views.get("panelordenes").vw.getWidth())));
views.get("btnreporteback").vw.setLeft((int)(0d));
views.get("btnreporteback").vw.setTop((int)(0d));
views.get("btnreporteback").vw.setHeight((int)((views.get("panelordentitulo").vw.getHeight())));
views.get("btnreporteback").vw.setWidth((int)((15d / 100 * width)));
views.get("label1").vw.setLeft((int)((views.get("btnreporteback").vw.getLeft() + views.get("btnreporteback").vw.getWidth())));
views.get("label1").vw.setHeight((int)((views.get("panelordentitulo").vw.getHeight())));
views.get("label1").vw.setWidth((int)((views.get("panelordentitulo").vw.getWidth())-(views.get("label1").vw.getLeft())));
views.get("label1").vw.setTop((int)(0d));
views.get("label2").vw.setTop((int)((views.get("panelordentitulo").vw.getTop() + views.get("panelordentitulo").vw.getHeight())+(2d / 100 * height)));
views.get("label2").vw.setHeight((int)((7d / 100 * height)));
views.get("label2").vw.setWidth((int)((20d / 100 * width)));
views.get("label2").vw.setLeft((int)((3d / 100 * width)));
views.get("panel1").vw.setTop((int)((views.get("label2").vw.getTop())));
views.get("panel1").vw.setLeft((int)((views.get("label2").vw.getLeft() + views.get("label2").vw.getWidth())));
views.get("panel1").vw.setHeight((int)((views.get("label2").vw.getHeight())));
views.get("panel1").vw.setWidth((int)((70d / 100 * width)));
views.get("date").vw.setLeft((int)(0d));
views.get("date").vw.setTop((int)(0d));
views.get("date").vw.setWidth((int)((views.get("panel1").vw.getWidth())));
views.get("date").vw.setHeight((int)((views.get("panel1").vw.getHeight())));
views.get("chordenesabiertas").vw.setTop((int)((views.get("panel1").vw.getTop() + views.get("panel1").vw.getHeight())+(0.5d / 100 * height)));
views.get("chordenesabiertas").vw.setWidth((int)((90d / 100 * width)));
views.get("chordenesabiertas").vw.setLeft((int)((views.get("label2").vw.getLeft())));
views.get("chordenesabiertas").vw.setHeight((int)((5d / 100 * height)));
views.get("label3").vw.setTop((int)((views.get("chordenesabiertas").vw.getTop() + views.get("chordenesabiertas").vw.getHeight())+(2d / 100 * height)));
views.get("label3").vw.setHeight((int)((5d / 100 * height)));
views.get("label3").vw.setLeft((int)((views.get("chordenesabiertas").vw.getLeft())));
views.get("label3").vw.setWidth((int)((views.get("panelordenes").vw.getWidth())-(views.get("label3").vw.getLeft())*2d));
views.get("panel2").vw.setTop((int)((views.get("label3").vw.getTop() + views.get("label3").vw.getHeight())));
views.get("panel2").vw.setWidth((int)((views.get("label3").vw.getWidth())));
views.get("panel2").vw.setHeight((int)((60d / 100 * height)));
views.get("panel2").vw.setLeft((int)((views.get("label3").vw.getLeft())));
views.get("listviewordenes").vw.setLeft((int)(0d));
views.get("listviewordenes").vw.setWidth((int)((views.get("panel2").vw.getWidth())));
views.get("listviewordenes").vw.setHeight((int)((views.get("panel2").vw.getHeight())));
views.get("listviewordenes").vw.setTop((int)(0d));
views.get("btnbuscarreporte").vw.setLeft((int)((views.get("panel2").vw.getLeft())));
views.get("btnbuscarreporte").vw.setHeight((int)((7d / 100 * height)));
views.get("btnbuscarreporte").vw.setTop((int)((views.get("panel2").vw.getTop() + views.get("panel2").vw.getHeight())+(2d / 100 * height)));
views.get("btnbuscarreporte").vw.setWidth((int)((views.get("panel2").vw.getWidth())));

}
}

File: mobile_app/app_ios/Objects/src/B4iProject/LS_frmreporteordenes.m

#import "iCore.h"
@interface LS_frmreporteordenes:NSObject
@end

@implementation LS_frmreporteordenes

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[B4ILayoutBuilder scaleAll:views :width :height];
[views get:@"label2"].Top = (int)((1 / 100.0 * height));
[views get:@"label2"].Height = (int)((5 / 100.0 * height));
[views get:@"label2"].Width = (int)((50 / 100.0 * width));
[views get:@"label2"].Left = (int)((3 / 100.0 * width));
[views get:@"panel1"].Top = (int)(([views get:@"label2"].Top + [views get:@"label2"].Height));
[views get:@"panel1"].Left = (int)(([views get:@"label2"].Left));
[views get:@"panel1"].Height = (int)((15 / 100.0 * height));
[views get:@"panel1"].Width = (int)((100 / 100.0 * width)-([views get:@"panel1"].Left)*2.0);
[views get:@"date"].Left = (int)(0.0);
[views get:@"date"].Top = (int)(0.0);
[views get:@"date"].Width = (int)(([views get:@"panel1"].Width));
[views get:@"date"].Height = (int)(([views get:@"panel1"].Height));
[views get:@"chordenesabiertas"].Top = (int)(([views get:@"panel1"].Top + [views get:@"panel1"].Height)+(1.55 / 100.0 * height));
[views get:@"chordenesabiertas"].Width = (int)((15 / 100.0 * width));
[views get:@"chordenesabiertas"].Left = (int)(([views get:@"label2"].Left));
[views get:@"chordenesabiertas"].Height = (int)((5 / 100.0 * height));
[views get:@"label1"].Left = (int)(([views get:@"chordenesabiertas"].Left + [views get:@"chordenesabiertas"].Width));
[views get:@"label1"].Top = (int)(([views get:@"chordenesabiertas"].Top));
[views get:@"label1"].Height = (int)(([views get:@"chordenesabiertas"].Height));
[views get:@"label1"].Width = (int)((100 / 100.0 * width)-([views get:@"label1"].Left));
[views get:@"label3"].Top = (int)(([views get:@"chordenesabiertas"].Top + [views get:@"chordenesabiertas"].Height)+(0.5 / 100.0 * height));
[views get:@"label3"].Height = (int)((5 / 100.0 * height));
[views get:@"label3"].Left = (int)(([views get:@"chordenesabiertas"].Left));
[views get:@"label3"].Width = (int)((100 / 100.0 * width)-([views get:@"label3"].Left)*2.0);
[views get:@"panel2"].Top = (int)(([views get:@"label3"].Top + [views get:@"label3"].Height));
[views get:@"panel2"].Width = (int)(([views get:@"label3"].Width));
[views get:@"panel2"].Height = (int)((57 / 100.0 * height));
[views get:@"panel2"].Left = (int)(([views get:@"label3"].Left));
[views get:@"btnbuscarreporte"].Left = (int)(([views get:@"panel2"].Left));
[views get:@"btnbuscarreporte"].Height = (int)((7 / 100.0 * height));
[views get:@"btnbuscarreporte"].Top = (int)(([views get:@"panel2"].Top + [views get:@"panel2"].Height)+(1 / 100.0 * height));
[views get:@"btnbuscarreporte"].Width = (int)(([views get:@"panel2"].Width));

}
@end